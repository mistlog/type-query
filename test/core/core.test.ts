import { getResultCode } from "../common";

describe("core", () => {
    test("copy type with utils .use and .type", () => {
        const name = "basic";
        const code = getResultCode(name);
        expect(code).toMatchSnapshot();
    })
})