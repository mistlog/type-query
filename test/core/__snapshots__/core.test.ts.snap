// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core copy type with utils .use and .type 1`] = `
"type Todo = {
  title: string;
  description: string;
  completed: boolean;
};
type TodoPreview = {
  title: string;
  description: string;
  completed: boolean;
};"
`;
